pkg/operator/target_config_reconciler.go

```diff
@@ -311,6 +311,11 @@ func (c TargetConfigReconciler) sync() error {
 		return err
 	}
 
+	if err := c.manageNetworkPolicies(ownerReference); err != nil {
```

Comment:
We would need to change the namespace if a user installs this in a different namespace from `openshift-kueue-operator`.